content-repo/gen_top_contrib.py

```diff
@@ -293,10 +354,31 @@ def main():
     args = parser.parse_args()
     contrib_target = args.target + '/top-contributors.md'
     try:
-        response = get_inner_pr_request()
-        users_list = get_contributors_users(response)
+        service_account_file = tempfile.NamedTemporaryFile(delete=False, mode='w', suffix='.json')
+        json.dump(json.loads(SERVICE_ACCOUNT), service_account_file)
+        service_account_file.flush()
```

Comment:
Let's move this into `get_contributors_file_from_bucket` function

Reply:
This I need for two functions - need it for the update file also.
I can move it and return the file as well, but I think that this part should stay in the main function

Reply:
Good point let's move it into a separate function that will create the file and will return the object to be used later on. The idea (and that is also true for my other comments) that main functions should be very straightforward and hence - relatively short, clear, and readable.